packages/arb-token-bridge-ui/src/util/TokenWithdrawalUtils.ts

```diff
@@ -0,0 +1,32 @@
+import { Erc20Bridger } from '@arbitrum/sdk'
+import { Provider } from '@ethersproject/providers'
+import { BigNumber } from 'ethers'
+import { GasEstimates } from '../hooks/arbTokenBridge.types'
+
+export async function withdrawTokenEstimateGas({
+  amount,
+  address,
+  erc20L1Address,
+  l2Provider
+}: {
+  amount: BigNumber
+  address: string
+  erc20L1Address: string
+  l2Provider: Provider
+}): Promise<GasEstimates> {
+  const erc20Bridger = await Erc20Bridger.fromProvider(l2Provider)
+  const estimatedL1Gas = BigNumber.from(160_000)
+
+  const withdrawalRequest = await erc20Bridger.getWithdrawalRequest({
+    amount,
+    destinationAddress: address,
+    erc20l1Address: erc20L1Address,
```

Comment:
i saw that it was like this before but we can probably just shorthand it in the future

Reply:
Eslint is able to enforce it

https://eslint.org/docs/latest/rules/object-shorthand

Reply:
There is a difference in casing here between `erc20l1Address` and `erc20L1Address` 🤦